src/main.js

```diff
@@ -17,3 +20,15 @@ var currentCover;
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
+
+function randomEverything(covers, titles, descriptors) {
+  var x = getRandomIndex(covers)
+  var y = getRandomIndex(titles)
+  var z = getRandomIndex(descriptors);
+  var zz = getRandomIndex(descriptors);
```

Comment:
I love it! Some refactoring, I would suggest instead of x, y, z, zz we do "coverIndex", "titleIndex" etc just so it's a little easier to read